locations/spiders/lidl_at.py

```diff
@@ -0,0 +1,27 @@
+import re
+
+from locations.hours import DAYS_HU, OpeningHours, sanitise_day
+from locations.spiders.lidl_gb import LidlGBSpider
+from locations.storefinders.virtualearth import VirtualEarthSpider
+
+
+class LidlHUSpider(VirtualEarthSpider):
+    name = "lidl_at"
+    item_attributes = LidlGBSpider.item_attributes
+
+    dataset_id = "d9ba533940714d34ac6c3714ec2704cc"
+    dataset_name = "Filialdaten-AT/Filialdaten-AT"
+    key = "Ailqih9-jVv2lUGvfCkWmEFxPjFBNcEdqZ3lK_6jMMDDtfTYu60SwIaxs32Wtik2"
+
+    def parse_item(self, item, feature, **kwargs):
+        item["name"] = feature["ShownStoreName"]
+
+        item["opening_hours"] = OpeningHours()
+        for day, start_time, end_time in re.findall(
+            r"(\w+) (\d{2}:\d{2})-(\d{2}:\d{2})",
+            feature["OpeningTimes"],
+        ):
+            if day := sanitise_day(day, DAYS_AT):
```

Comment:
We don't have a `DAYS_AT` yet.

But checkout `hours.py` if you want to make one, the input looks quite good:
```python
{'days/Di': 251,
 'days/Do': 251,
 'days/Fr': 251,
 'days/Mi': 251,
 'days/Mo': 251,
 'days/Sa': 251,
 'days/So': 4
}```